chart/templates/secrets.yaml

```diff
@@ -44,9 +44,19 @@ stringData:
   SSLMODE: {{ $sslmode | quote }}
   DB_URL:  {{ (print "postgres://" $user ":" $password "@" $postgresHost "/" $database "?sslmode=" $sslmode ) | quote }}
   DATABASE: {{ $database | quote }}
-
 {{- end }}
 
+{{- if .Values.adminPassword.secretKeyRef.create }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.adminPassword.secretKeyRef.name }}
+type: Opaque
+stringData:
+  password: {{ randAlphaNum 12 }}
```

Comment:
This would change the password on each reconciliation - we need to lookup the existing password, and then default to random like we do for db secrets